packages/loader/container-loader/src/container.ts

```diff
@@ -836,11 +839,29 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this._resolvedUrl = resolvedUrl;
             await this.connectStorageService();
 
-            // This we can probably just pass the storage service to the blob manager - although ideally
-            // there just isn't a blob manager
+            // upload blobs here (NYI)
+
+            // post summary here
+            if (hasAttachmentBlobs) {
+                const appSummary: ISummaryTree = this.context.createSummary();
+                const protocolSummary = this.captureProtocolSummary();
+                summary = combineAppAndProtocolSummary(appSummary, protocolSummary);
+
+                this._attachState = AttachState.Attaching;
+                // this.emit("attaching");
+                this.context.notifyAttaching();
+
+                await this.storageService.uploadSummaryWithContext(summary, {
+                    referenceSequenceNumber: 0,
+                    ackHandle: undefined,
+                    proposalHandle: undefined,
+                });
+
+                assert(!hasAttachmentBlobs, "attaching container with blobs is not yet implemented");
```

Comment:
I think you can move it at the top after you initialize this variable as it seems useless here.